packages/tasks_panel/src/utils/task-serializer.ts

```diff
@@ -1,5 +1,70 @@
 import { ConfiguredTask } from "@sap_oss/task_contrib_types";
+import { randomBytes } from "crypto";
+import escapeStringRegexp = require("escape-string-regexp");
+import { filter, map } from "lodash";
+import { getConfiguredTasksFromCache } from "../../src/services/tasks-provider";
+import { ConfigurationTarget, TaskDefinition, Uri, commands, languages, window, workspace } from "vscode";
 
 export function serializeTask(task: ConfiguredTask): string {
   return JSON.stringify(task);
 }
+
+export function generateUniqueCode(): string {
+  function generateGroup(): string {
+    return randomBytes(2).toString("hex").toUpperCase();
+  }
+  return `${generateGroup()}-${generateGroup()}`;
```

Comment:
what is the difference between generating 4 bytes of random data
and generating 2 bytes twice and linking them with "-" ?